09-sorting-algorithms/lib/quicksort.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+let quicksort = (array) => {
+  if (!array) {
+    throw new Error('ERROR: you must pass in an array');
+  }
+  if (array.length <= 1) {
+    return array;
+  } else {
+    let pivot = array.pop();
+    let left = [];
+    let right = [];
+    let newArray = [];
```

Comment:
newArray isn't a great name. Try to avoid using `new` and `old` with variable names, unless context is actually talking about age. Instead, this could have been called `result` since that is how it gets used.